src/LibraryManager.Vsix/LibraryManagerPackage.cs

```diff
@@ -43,18 +47,27 @@ namespace Microsoft.Web.LibraryManager.Vsix
 
     internal sealed class LibraryManagerPackage : AsyncPackage
     {
+        [Import]
+        ILibraryCommandService libraryCommandService { get; set; }
+
         protected override async Tasks.Task InitializeAsync(CancellationToken cancellationToken, IProgress<ServiceProgressData> progress)
         {
-            if (await GetServiceAsync(typeof(IMenuCommandService)).ConfigureAwait(false) is OleMenuCommandService commandService)
+            await ThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();
+
+            IComponentModel componentModel = await GetServiceAsync(typeof(SComponentModel)) as IComponentModel;
+            componentModel.DefaultCompositionService.SatisfyImportsOnce(this);
+
+            if (GetService(typeof(IMenuCommandService)) is OleMenuCommandService commandService)
```

Comment:
>GetService [](start = 16, length = 10)

I assume GetServiceAsync isn't going to work here?

Reply:
Fixed and cleaned up